examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -106,28 +108,43 @@ async def raise_if_error_async(awaitable_call):
                         task=task, attribute=nidaqmx_types.TASK_ATTRIBUTE_NUM_CHANS)))
 
             number_of_channels = response.value
-
             async def read_data():
-                async for every_n_samples_response in every_n_samples_stream:
-                    await raise_if_error(every_n_samples_response)
-                    read_response: nidaqmx_types.ReadAnalogF64Response = await raise_if_error_async(
-                        client.ReadAnalogF64(
-                            nidaqmx_types.ReadAnalogF64Request(
-                                task=task,
-                                num_samps_per_chan=100,
-                                fill_mode=nidaqmx_types.GroupBy.GROUP_BY_GROUP_BY_CHANNEL,
-                                array_size_in_samps=number_of_channels * 100)))
-
-                    print(
-                        f"Acquired {len(read_response.read_array)} samples",
-                        f"({read_response.samps_per_chan_read} samples per channel)")
-                    print("Read Data:", read_response.read_array[:10])
+                samps_per_chan_read = 0
+
+                try:
+                    async for every_n_samples_response in every_n_samples_stream:
+                        await raise_if_error(every_n_samples_response)
+                        read_response: nidaqmx_types.ReadAnalogF64Response = await raise_if_error_async(
```

Comment:
Good question, I tend to use a lot fewer types in examples, but I do add them sometimes. Especially when I'd have a hard time writing it without the types. And, yeah, I think it's an easier choice for an asyncio example where people using it would need to be fairly aware of modern python.

This case works even better with a `TypeParam` on `raise_if_error_async` to pass through the type in mypy but I don't think we want a tutorial on that in our examples.